cross-chain/arbitrum/contracts/ArbitrumWormholeGateway.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
```

Comment:
Should we add [ASCII art that we usually do in out contracts](https://github.com/keep-network/tbtc-v2/blob/5d6232bb50660ae04f5c4af3fc2f53bc989ab251/solidity/contracts/bridge/Bridge.sol#L3-L14)?

Reply:
I don't know actually. This project will be integrated with the Wormhole through their contracts. @pdyraga can we add our T ASCII or leave it as is?

Reply:
It's gone now.